src/components/specific/app/go-back-button/GoBackButton.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <BIMDataButton
-    data-test="btn-back"
+    data-test-id="go-back-button"
```

Comment:
It's `data-test` on the viewer and `data-test-id` hare. We may want to uniformize them

Reply:
Yes, I agree.
I'm more comfortable with `data-test-id` since I also use another data attribute to store test related data: `data-test-param`.